news/2023/2023-10-08-osu-mania-world-cup-2023.md

```diff
@@ -0,0 +1,165 @@
+
+Just as they had felt out the weaknesses from each other the week before, United States stuck to their overall dominance on rice, claiming points on them while trying their best to steal breakpoints from rice-dominant hybrids and LNs by virtue of consistency. During this grand finale of a match, a couple of break points were lost by the Koreans to the Americans, including [Kyuusei Argyros](https://osu.ppy.sh/beatmapsets/2062249#mania/4311953) and [Angel Echo](https://osu.ppy.sh/beatmapsets/2062240#mania/4311937), all which led down to the game tilting in the favor of the United States.
+
+With a definitive 32,000 score difference on [Shinsha](https://osu.ppy.sh/beatmapsets/2062237#mania/4311929), United States claims a 7-3 win, bringing the champion title home to the Americans once more, and with the Koreans claiming the silver medal after a tough struggle to the top of the summit. Congratulations to both teams!
+
+Along the way, we paid witness to some of the most impressively consistent and clean scores on the toughest MWC pool thus far:
+
+- cafesito’s very impressive 995k score in [MULA](https://osu.ppy.sh/beatmapsets/2062251#mania/4311956), which could have been seen as challenging for even the best SV players;
+- yz1155’s impressive Full Combos on [Selenadia](https://osu.ppy.sh/beatmapsets/2062245#mania/4311944) and [Kyuusei Argyros](https://osu.ppy.sh/beatmapsets/2062249#mania/4311953);
+- Nepijin’s 9x200 score on [Angel Echo](https://osu.ppy.sh/beatmapsets/2062240#mania/4311937), 996k on [Shinsha](https://osu.ppy.sh/beatmapsets/2062237#mania/4311929), and 99.3% on the tiebreaker showcase for [Parallel Universe Shifter](https://osu.ppy.sh/beatmapsets/2062263#mania/4312004).
+
+## Interviews
+
+We've interviewed a couple active and well-known players in the tournament community.
+
+### myucchii
+
+*Hello myucchii! Thank you so much for taking the time to do this quick interview with us. Let’s just cut straight to the chase. First things first: let’s talk about the tiebreaker between Chile and Vietnam. Walk us through what was going on in your mind, how you felt before, during, and after.*
+
+> **myucchii:** The match against Vietnam was overall very stressful to me. Even though I was feeling good, and the team was performing very well, we weren't getting the points we were expecting to get in the match. I was also really nervous because this was my first time getting into the semifinals. We ended up getting to the tiebreaker, and I never checked the scoreboard at any point; I was only focused on comboing and trying my hardest to get my team to win. The moment I saw the final message from the referee, saying that we won the match, was probably one of the best moments I've ever experienced in my entire osu!mania career as a player, to the point where I cried because of how happy I was. Every time I scroll through my maps in the osu! menu and see HALL, I just think of that moment.
```

Comment:
```suggestion
> **myucchii:** The match against Vietnam was overall very stressful to me. Even though I was feeling good, and the team was performing very well, we weren't getting the points we were expecting to get in the match. I was also really nervous because this was my first time getting into the semifinals. We ended up getting to the tiebreaker, [HALL](https://osu.ppy.sh/beatmapsets/2055332#mania/4294726), and I never checked the scoreboard at any point; I was only focused on comboing and trying my hardest to get my team to win. The moment I saw the final message from the referee, saying that we won the match, was probably one of the best moments I've ever experienced in my entire osu!mania career as a player, to the point where I cried because of how happy I was. Every time I scroll through my maps in the osu! menu and see HALL, I just think of that moment.
```